src/Sylius/Behat/Context/Setup/Api/CartContext.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
```

Comment:
missing doc bloc

And I am not sure that this context is necessary, Setup should not depend on UI or API context.

Reply:
I would like to add @GSadee @lchrusciel for our little conversation 🍵

Reply:
I've noticed the same thing in a comment below, @arti0090 could you tell us why you added this context and changed the current setup cart context? Because I agree with @AdamKasp that the setup shouldn't depend on UI or API and we should dispatch proper commands there.

Reply:
By utilizing old setup/CartContext there are few problems:

1.  We are mixing contexts so the behat doesn't f.e. choose correct channel and then when going from setup to api doesn't set this channel to order
2. Before creating order we need to choose on which hostname (channel for API) we are operating, setting this hostname is crucial for next steps
3. In behat we have those two steps `Given I changed my current channel to "United States"` and `And I have product "PHP T-Shirt" in the cart` when I created them with setup/CartContext on the execution there was an exception 
![Zrzut ekranu 2020-11-27 o 09 06 49](https://user-images.githubusercontent.com/35863747/100426267-de6c5e80-3090-11eb-934a-c39066d8b59b.png)

Reply:
I would move back setup to command dispatching only and refactor of pickup cart command, so we can add channel to it